### ISSUE TITLE
How to predict more than 1 samples at once?

### ISSUE THREAD

**Issue by vanzytay** [Conchylicultor/DeepQA#42]
Hi,

Thanks for the amazing repo!

I noticed you only do singlePredict even when predicting a list of test data. I tried to pass in batch size>1 for testing and got `tensorflow.python.framework.errors.InvalidArgumentError: ConcatOp : Dimensions of inputs should match: shape[0] = [1,100] vs. shape[1] = [100,300]`

I wonder if you know any idea how to overcome this?

Thanks in advance! 

Have a good day!

**Comment by Conchylicultor** [comment]
Indeed, interactive and testing mode only generate one sentence at a time.
It should be possible to add minibatch support by modifying the
`sentence2enco` and `deco2sentence` functions in `textdata.py`. For your
exact exception, it's difficult to say without having the full stack trace.
